### PR TITLE
Add back Ruby 2.2 support for Rails 5

### PR DESCRIPTION
**Pull request by JonRowe**

### CODE REVIEW

**Comment by JonRowe** [comment]
@benoittgt I've temporarily suppressed our phantom error as I'd like to do a release to resolve this issue in particular

**Comment by benoittgt** [comment]
No problem Jon.

**Comment by benoittgt** [comment]
If you need some help on this @JonRowe

**Comment by JonRowe** [comment]
I've just been playing whackamole with the Travis build @benoittgt :joy: I think it'll pass now, care to re-review?

File: .travis.yml [hunk 1]
```diff
@@ -77,6 +77,11 @@ matrix:
       env: RAILS_VERSION='~> 5.2.0'
     - rvm: 2.3.8
       env: RAILS_VERSION='~> 5.2.0'
+    - rvm: 2.2.10
+      env: RAILS_VERSION='~> 5.2.0'
+      allow_failure: true
+    - rvm: 2.2.10
+      env: RAILS_VERSION='5-2-stable'
```
Comment (benoittgt) [inline .travis.yml:84]: Why do we want to try stable and `~> 5.2.0`?

File: spec/rspec/rails/example/view_example_group_spec.rb [hunk 2]
```diff
@@ -234,17 +236,29 @@ def _view; end # Stub method
...
             allow(view).to receive(:render) { :value }
             run_count += 1
           end
           group = RSpec::Core::ExampleGroup.describe 'a view', type: :view do
-            specify { true }
+            specify { expect(true).to eq true }
           end
-          group.run
+          group.run(failure_reporter)
+          expect(failure_reporter.exceptions).to eq []
```
Comment (benoittgt) [inline spec/rspec/rails/example/view_example_group_spec.rb:261]: I do not understand why we need the `FailureReporter` anymore?

File: .travis.yml (hunk 1 above)
Comment (JonRowe) [reply .travis.yml:84]: `5-2-stable` is `2.2.10` compatible, it contains an unreleased bugfix for Ruby 2.2, `~> 5.2.0` is included but will fail, eventually when patched the pending spec will fail this build and we can remove the `5-2-stable` the same as the other versions.

File: spec/rspec/rails/example/view_example_group_spec.rb (hunk 2 above)
Comment (JonRowe) [reply spec/rspec/rails/example/view_example_group_spec.rb:261]: Its included for future debugging, we've had this spec fail before and its really hard to see why without this pattern, I'm going to use it in other places for the same reason.

File: .travis.yml (hunk 1 above)
Comment (benoittgt) [reply .travis.yml:84]: Ok. Thanks!

File: spec/rspec/rails/example/view_example_group_spec.rb (hunk 2 above)
Comment (benoittgt) [reply spec/rspec/rails/example/view_example_group_spec.rb:261]: Nice! Maybe we could add a little bit of documentation on the top of file or somewhere?

**Pull request merged**